source/octod/api/projects.d

```diff
@@ -17,11 +17,18 @@
 module octod.api.projects;
 
 import octod.core;
+import octod.media;
 import octod.api.common;
 
 import vibe.data.json;
 
 /**
+    Project API is experimental, thus relevant requests have to use
+    specific media type.
+ **/
+enum ProjectMediaType = "application/vnd.github.inertia-preview+json";
```

Comment:
Maybe is to address in another PR, but for me is weird that the user of the library can't decide if they are using a preview API or not. I have the feeling users should say somewhere which API version to use, and based on that certain objects/methods should be available or not.

For me is weird that using a non-stable API is hidden from the user.